lib/controller.rb

```diff
@@ -7,5 +7,13 @@
 class WeekThreeAssessment < Sinatra::Base
 
   # Your code will go here.
+get '/' do
+  @name = 'Amanda'
+  erb :index
+end
+
+get '/name/:name' do
+  @user = User.create(name: "#{params[:name]}")
+end
```

Comment:
Please mind code formatting in this file.

Reply:
I'm not 100% sure how you want me to format this.